LICENSE

```diff
@@ -198,4 +198,108 @@
+      java/fury-core/src/main/java/org/apache/fury/util/Platform.java
+      java/fury-format/src/main/java/org/apache/fury/format/vectorized/ArrowWriter.java
+
+* commons-io (https://github.com/apache/commons-io)
+    Files:
+      java/fury-core/src/main/java/org/apache/fury/io/ClassLoaderObjectInputStream.java
+
+
+* fast-serialization (https://github.com/RuedigerMoeller/fast-serialization)
+    Files:
+      java/benchmark/src/main/java/org/apache/fury/benchmark/Identity2IdMap.java
+
+* ray (https://github.com/ray-project/ray)
+    Files:
+      ci/format.sh
+      bazel/fury_deps_setup.bzl
+      bazel/grpc-cython-copts.patch
+      bazel/grpc-python.patch
+
+* tensorflow (https://github.com/tensorflow/tensorflow)
+    Files:
+      bazel/arrow/pyarrow_configure.bzl
+
+
+================================================================
+BSD-3-Clause licenses
+================================================================
+The following components are provided under the BSD-3-Clause License. See project link for details.
+The text of each license is also included in licenses/LICENSE-[project].txt.
+
+* kryo (https://github.com/EsotericSoftware/kryo)
+    Files:
+      java/benchmark/src/main/java/org/apache/fury/benchmark/data/Data.java
+      java/benchmark/src/main/java/org/apache/fury/benchmark/data/Image.java
+      java/benchmark/src/main/java/org/apache/fury/benchmark/data/Media.java
+      java/benchmark/src/main/java/org/apache/fury/benchmark/data/MediaContent.java
+      java/benchmark/src/main/java/org/apache/fury/benchmark/data/Sample.java
+      java/benchmark/src/main/java/org/apache/fury/benchmark/state/BufferType.java
+      java/benchmark/src/main/java/org/apache/fury/benchmark/state/KryoState.java
+      java/fury-core/src/main/java/org/apache/fury/collection/FuryObjectMap.java
+      java/fury-core/src/main/java/org/apache/fury/collection/IdentityMap.java
+      java/fury-core/src/main/java/org/apache/fury/collection/IdentityObjectIntMap.java
+      java/fury-core/src/main/java/org/apache/fury/collection/LongMap.java
+      java/fury-core/src/main/java/org/apache/fury/collection/ObjectIntMap.java
+      java/fury-core/src/main/java/org/apache/fury/type/Generics.java
+      java/fury-core/src/test/java/org/apache/fury/type/GenericsTest.java
+
+* v8 (https://chromium.googlesource.com/v8/v8.git)
+    Files:
+      javascript/packages/hps/src/v8-fast-api-calls.h
+
+
+================================================================
+MIT licenses
+================================================================
+The following components are provided under the MIT License. See project link for details.
+The text of each license is also included in licenses/LICENSE-[project].txt
+
+* murmurHash3.js (https://github.com/karanlyons/murmurHash3.js)
+    Files:
+      javascript/packages/fury/lib/murmurHash3.ts
+
+* cython-abseil-sample (https://github.com/messense/cython-abseil-sample)
```

Comment:
we don't have a cython-abseil-sample file in our licenses die

Reply:
You mean add this https://github.com/messense/cython-abseil-sample/blob/master/LICENSE to `license-abseil.txt`?

Reply:
yes - the licenses dir needs a license file for this code

Reply:
added LICENSE-cython-abseil-example.txt